depthai/plugin.cpp

```diff
@@ -0,0 +1,344 @@
+
+#include <cstdio>
+#include <iostream>
+
+#include "opencv2/opencv.hpp"
+
+// Inludes common necessary includes for development using depthai library
+#include "depthai/depthai.hpp"
+
+// ILLIXR includes
+#include "common/threadloop.hpp"
+#include "common/switchboard.hpp"
+#include "common/data_format.hpp"
+
+using namespace ILLIXR;
+
+class depthai : public plugin {
+public:
+    depthai(std::string name_, phonebook* pb_)
+        : plugin{name_, pb_}
+        , sb{pb->lookup_impl<switchboard>()}
+        , _m_imu_cam{sb->get_writer<imu_cam_type>("imu_cam")}
+        , _m_rgb_depth{sb->get_writer<rgb_depth_type>("rgb_depth")}
+        //Initialize DepthAI pipeline and device 
+        , p{createCameraPipeline()}
+        , d{p}
```

Comment:
Maybe we can use a variables that a a little more descriptive but still short (e.g., `cp`, or `dev`)